### ISSUE TITLE
🚀 Feature: Enable CrewAI Observability

### ISSUE THREAD

**Issue by gyliu513** [traceloop/openllmetry#2339]
### Which component is this feature for?

All Packages

### 🔖 Feature description

Enable instrumentation for crewai

### 🎤 Why is this feature needed ?

openllmetry does not work crewai with litellm version, it cannot track the model api calls.

### ✌️ How do you aim to achieve this?

Implement a crewai instrumentation library.

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### Are you willing to submit PR?

Yes I am willing to submit a PR!

**Comment by gyliu513** [comment]
@galkleinman are you working for crweai instrumentation now? There is a branch from you at https://github.com/traceloop/openllmetry/tree/gk/crewai-instrumentation

@adharshctr @nirga ^^

**Comment by galkleinman** [comment]
@gyliu513 I started working on it as a weekend project couple of weeks ago. You can take it from here and take some inspiration from that branch (it's actually working nice in high-level). Pay attention to the fact CrewAI emitting otel spans themselves as well...

**Comment by gyliu513** [comment]
ok, thanks @galkleinman , we will continue your work here

**Comment by adharshctr** [comment]
Hi @galkleinman 
 I just  checkout to your gk/crewai-instrumentation branch and tried building and installing the crewai instrumentation via
 nx run opentelemetry-instrumentation-crewai:build  
  nx run opentelemetry-instrumentation-crewai:install
  
  How to install traceloop sdk and test crew instrumentation  from  gk/crewai-instrumentation branch?
  
 Can you please share me the steps?

**Comment by nirga** [comment]
@adharshctr @gyliu513 are you still working on this?

**Comment by gyliu513** [comment]
@nirga Yes, we have a prototype ready based on @galkleinman 's work, will create a PR soon. (@adharshctr is on holiday, the PR will probably be created in early Jan), hope it is OK, thanks!

**Comment by detunjiSamuel** [comment]
Hey @gyliu513 , are you still working on this ?

**Comment by gyliu513** [comment]
Hi @detunjiSamuel we have a PR here https://github.com/traceloop/openllmetry/pull/2489 under review, thanks!

**Comment by nirga** [comment]
I think this is solved with #2489
